src/nock/reporter.ts

```diff
@@ -0,0 +1,108 @@
+import { NonPrimitive } from '@superfaceai/one-sdk/dist/internal/interpreter/variables';
+import createDebug from 'debug';
+import { join as joinPath } from 'path';
+
+import { exists, readFileQuiet, readFilesInDir, rimraf } from '../common/io';
+import { OutputStream } from '../common/output-stream';
+import {
+  TestAnalysis,
+  TestCoverageBase,
+  TestingReturn,
+} from '../superface-test.interfaces';
+import { AnalysisResult } from './analyzer';
+
+const DEFAULT_COVERAGE_PATH = 'coverage';
+
+const debug = createDebug('superface:testing:reporter');
+
+export class Reporter {
+  // TODO: hash this or collect this, so that `collect` function can add information about test result
+  static async save({
```

Comment:
question: Is there any value to have it as class with static functions instead of simple function?

Reply:
Nope, not yet 👍🏻